//
// Academic License - for use in teaching, academic research, and meeting
// course requirements at degree granting institutions only.  Not for
// government, commercial, or other organizational use.
//
// abcefg_T_A.h
//
// Code generation for function 'abcefg_T_A'
//

#ifndef ABCEFG_T_A_H
#define ABCEFG_T_A_H

// Include files
#include "rtwtypes.h"
#include <cstddef>
#include <cstdlib>

// Function Declarations
void abcefg_T_A(double V_init, double A_init, double A_wayp, double V_max,
                double A_max, double A_min, double J_max, double J_min,
                double T, double t[7]);

#endif
// End of code generation (abcefg_T_A.h)
